client/src/components/CreateAssignmentView.js

```diff
@@ -0,0 +1,77 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import React, { useState }  from 'react';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import Assignment from './create-assignment/Assignment';
+import subAssignmentServices from '../services/assignments';
```

Comment:
Could this be imported just as assignmentServices? Since it also contains a service for removing an assignment etc. Same with the other files